packages/integration-test-app/runner/lib/IntegrationTestRunner.ts

```diff
@@ -136,7 +136,7 @@ export default class IntegrationTestRunner {
 function failWithoutContext(message: string) {
   const err = new Error(message);
   err.stack = '\n';
-  fail(err);
+  throw err;
```

Comment:
Change made to replace `fail()` call which appears to no longer be available: https://github.com/jestjs/jest/issues/11698